memiavl/snapshot.go

```diff
@@ -176,8 +195,8 @@ func (snapshot *Snapshot) Node(index uint32) PersistedNode {
 }
 
 // Version returns the version of the snapshot
-func (snapshot *Snapshot) Version() uint64 {
-	return snapshot.version
+func (snapshot *Snapshot) Version() int64 {
```

Comment:
reason of using `int64` instead of `uint32`?

Reply:
fixed